can/network.yml

```
@@ -152,20 +153,38 @@ nodes:
 
         - BrakeData:
             id: 0x11
-            cycletime: 100
+            cycletime: 1
```

```suggestion
            cycletime: 10
```
1kHz for the brake data will saturate our bus.